lib/linguist/heuristics.yml

```diff
@@ -164,6 +164,10 @@ disambiguations:
   rules:
   - language: Gerber Image
     pattern: '^[DGMT][0-9]{2}\*(?:\r?\n|\r)'
+- extensions: ['.conf']
+  rules:
+  - language: BitBake
+    pattern: '^[A-Z_]+(([\[:][a-z])|(\s*[?+.]=))'
```

Comment:
Are you 100% sure this will only ever match BitBake files? If there's a chance this will match another language's files, it's best to not add this and the extension to `generic.yml`.

Reply:
I am not sure at all. On Linux people tend to use `.conf` extension for configuration files, which can be any format. I tried to make a restrictive regex, but the chance of having some false positive are great.

I will remove this extension from BitBake.